docs/_sass/_resets.scss

```diff
@@ -142,7 +142,7 @@ p {
 	padding: 5px 0;
 	border-radius: 6px;
 	position: absolute;
-        right: 43px;
+        right: -5px;
```

Comment:
So not only do we have this in CSS reset styles, it's duplicated in two places? What's going on with our styling system?

Reply:
this is really the only place it exists. this doc generates the styles above. (I had edited the wrong file first, so it stayed in the commit log)